### ISSUE TITLE
NoSuchMethod error

### ISSUE THREAD

**Issue by SlimeDog** [Rsl1122/Plan-PlayerAnalytics#514]
Spigot 1.12.2 latest
Plan 4.1.5-DEV1

Every second
```
[20:57:47] [Server thread/WARN]: [Plan] Task #213 for Plan v4.1.5-DEV generated an exception
java.lang.NoSuchMethodError: org.bukkit.Server.getTPS()[D
        at com.djrapitops.plan.system.tasks.bukkit.PaperTPSCountTimer.getTPS(PaperTPSCountTimer.java:16) ~[?:?]
        at com.djrapitops.plan.system.tasks.bukkit.BukkitTPSCountTimer.calculateTPS(BukkitTPSCountTimer.java:66) ~[?:?]
        at com.djrapitops.plan.system.tasks.bukkit.BukkitTPSCountTimer.addNewTPSEntry(BukkitTPSCountTimer.java:33) ~[?:?]
        at com.djrapitops.plan.system.tasks.TPSCountTimer.run(TPSCountTimer.java:36) ~[?:?]
        at com.djrapitops.plugin.task.RunnableFactory$1.run(RunnableFactory.java:38) ~[?:?]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[spigot-1.12.2.jar-2018-01-27-1906:git-Spigot-4bd94dc-015bda4]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:353) [spigot-1.12.2.jar-2018-01-27-1906:git-Spigot-4bd94dc-015bda4]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:739) [spigot-1.12.2.jar-2018-01-27-1906:git-Spigot-4bd94dc-015bda4]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot-1.12.2.jar-2018-01-27-1906:git-Spigot-4bd94dc-015bda4]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot-1.12.2.jar-2018-01-27-1906:git-Spigot-4bd94dc-015bda4]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot-1.12.2.jar-2018-01-27-1906:git-Spigot-4bd94dc-015bda4]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_151]
```

Ran without error on Plan 4.1.4. No other changes.

**Comment by Rsl1122** [comment]
Caused by https://github.com/Rsl1122/Abstract-Plugin-Framework/issues/8

**Comment by Rsl1122** [comment]
Updated the jar in Releases with this fix

**Comment by SlimeDog** [comment]
Confirmed fixed in second 4.1.5-DEV1.
